docs/systemd-with-workers/system/matrix-synapse-worker@.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Synapse %i
+
+# This service should be restarted when the synapse target is restarted.
+PartOf=matrix-synapse.target
+
+[Service]
+Type=notify
+NotifyAccess=main
+User=matrix-synapse
+WorkingDirectory=/var/lib/matrix-synapse
+EnvironmentFile=/etc/default/matrix-synapse
+ExecStart=/opt/venvs/matrix-synapse/bin/python -m synapse.app.generic_worker --config-path=/etc/matrix-synapse/homeserver.yaml --config-path=/etc/matrix-synapse/conf.d/ --config-path=/etc/matrix-synapse/workers/%i.yaml
```

Comment:
is synapse.app.generic_worker the new catch all for all workers? because
https://github.com/eMPee584/matrix-docker-ansible-deploy/commit/29e7a038e2d99a560efe93b5d5e4bafa13bbd05f#diff-73735c947beefd609096186b0eab2ec1

Reply:
.. ppsst.. @erikjohnston  .. ?

Reply:
closed PRs are not a good place to ask questions.